css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
```

Comment:
> sticking to the necessary title of introduction text above the widget.

Mmh. This makes an assumption on the client insertion strategy. I'd rather have us focus only on the inner boundaries of our widget, leaving insertion adjustments to the client.